code/modules/lighting/light_effect_cast.dm

```diff
@@ -66,14 +66,16 @@ var/light_post_processing = 1 // Use writeglobal to change this
 		alpha = initial(alpha)
 		animate(src, alpha = initial(alpha) - rand(30, 60), time = 2, loop = -1, easing = SINE_EASING)
 
-	for (var/mob/M in view(world.view, src))
-		M.check_dark_vision()
-
-	for(var/turf/T in view(2*light_range, src))
```

Comment:
Why is it okay for it to be affecting a turf outside its light range?

Reply:
It isn't, in retrospect.